lab-davide/gulpfile.js

```diff
@@ -0,0 +1,57 @@
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+const filepath = ['*.js', 'lab-davide/lib/*.js', 'lab-davide/test/*.js'];
+
+gulp.task('default', function() {
+  return gulp.src(filepath, {read: false})
+  .pipe(mocha({reporter: 'nyan'}));
+});
+
+gulp.task('eslint', function() {
+  return gulp.src(filepath)
+  .pipe(eslint(
+    {
```

Comment:
Break out your eslint configs into a .eslintrc file, do not include them in the body of your gulp file.